executor/common_linux.h

```diff
@@ -5695,24 +5702,18 @@ static void setup_swap()
 	unlink(SWAP_FILE);
 	// Zero-fill the file.
 	int fd = open(SWAP_FILE, O_CREAT | O_WRONLY | O_CLOEXEC, 0600);
-	if (fd == -1) {
+	if (fd == -1)
 		failmsg("swap file open failed", "file: %s", SWAP_FILE);
-		return;
```

Comment:
How could it help us if we run past this return in case of an invalid fd?

Reply:
fail/failmsg/exitf never return